src/odemis/util/test/util_test.py

```diff
@@ -396,54 +401,255 @@ def test_line_clipping(self):
 ENZEL_CONFIG = CONFIG_PATH + "sim/enzel-sim.odm.yaml"
 SPARC_CONFIG = CONFIG_PATH + "sim/sparc-sim.odm.yaml"
 
+
 class TestBackendStarter(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         # make sure initially no backend is running.
         if driver.get_backend_status() == driver.BACKEND_RUNNING:
-            test.stop_backend()
+            testing.stop_backend()
 
     @classmethod
     def tearDownClass(cls):
         # turn off everything when the testing finished.
         if driver.get_backend_status() == driver.BACKEND_RUNNING:
-            test.stop_backend()
+            testing.stop_backend()
 
     def test_no_running_backend(self):
         # check if there is no running backend
         backend_status = driver.get_backend_status()
         self.assertIn(backend_status, [driver.BACKEND_STOPPED, driver.BACKEND_DEAD])
         # run enzel
-        test.start_backend(ENZEL_CONFIG)
+        testing.start_backend(ENZEL_CONFIG)
         # now check if the role is enzel
         role = model.getMicroscope().role
         self.assertEqual(role, "enzel")
 
     def test_running_backend_same_as_requested(self):
         # run enzel backend
-        test.start_backend(ENZEL_CONFIG)
+        testing.start_backend(ENZEL_CONFIG)
         # check if the role is enzel
         role = model.getMicroscope().role
         self.assertEqual(role, "enzel")
         # run enzel backend again
-        test.start_backend(ENZEL_CONFIG)
+        testing.start_backend(ENZEL_CONFIG)
         # it should still be enzel.
         role = model.getMicroscope().role
         self.assertEqual(role, "enzel")
 
     def test_running_backend_different_from_requested(self):
         # run sparc backend
-        test.start_backend(SPARC_CONFIG)
+        testing.start_backend(SPARC_CONFIG)
         # check if the role is sparc
         role = model.getMicroscope().role
         self.assertEqual(role, "sparc")
         # now run another backend (enzel)
-        test.start_backend(ENZEL_CONFIG)
+        testing.start_backend(ENZEL_CONFIG)
         # check if the role now is enzel instead of sparc
         role = model.getMicroscope().role
         self.assertEqual(role, "enzel")
 
 
+class FindClosestTestCase(unittest.TestCase):
+
+    def test_simple(self):
+        """ Finding the closest point in a single list """
```

Comment:
It's not a hard rule but in general for one line docstrings we do not put spaces between the quotes and the text, so this one would become:
```
"""Finding the closest point in a single list"""
```

Feel free to leave it like you had it, or update it for all your test cases.